src/main/kotlin/com/pubnub/api/endpoints/objects/internal/ReturningCustom.kt

```diff
@@ -31,8 +31,17 @@ data class Include(
     }
 }
 
-@Deprecated("use Include", ReplaceWith("Include"))
-data class ReturningCustom @Deprecated("use Include", ReplaceWith("Include(includeCustom = includeCustom)")) constructor(
+@Deprecated(
+    "use IncludeQueryParam",
+    ReplaceWith("IncludeQueryParam", "com.pubnub.api.endpoints.objects.internal.IncludeQueryParam")
+)
+data class ReturningCustom @Deprecated(
+    "use IncludeParam",
```

Comment:
seems that should be "use IncludeQueryParam" instead of "use IncludeParam"

Reply:
yes, but I realised that we can actually remove this completely